kopykat-ksp/src/main/kotlin/at/kopyk/KopyKatProcessor.kt

```diff
@@ -29,14 +31,26 @@ internal class KopyKatProcessor(
   private val options: KopyKatOptions
 ) : SymbolProcessor {
 
+  @OptIn(KspExperimental::class)
   override fun process(resolver: Resolver): List<KSAnnotated> {
     resolver.getAllFiles().let { files ->
       if (files.none { it.hasGeneratedMarker() }) {
-        files.flatMap { it.allNestedDeclarations() }
-          .filter { it is KSClassDeclaration || it is KSTypeAlias }
-          .onEach { it.checkAnnotationMisuse() }
+        val declarations = files.flatMap { it.allNestedDeclarations() }
+
+        val classes = declarations
+          .filterIsInstance<KSClassDeclaration>()
+          .onEach { it.checkKnown() ; it.checkRedundantAnnotation() }
```

Comment:
Let's avoid semicolons 🙏 😅 
```sugestion
          .onEach { it.checkKnown() }
          .onEach { it.checkRedundantAnnotation() }
```

Reply:
Fine, although this will report errors in a different order compared to before

Reply:
Oh, I didn't think of that 🙈 How bad would it be for devX?
Happy to put them on separate lines in the same onEach.

Reply:
It shouldn’t really matter, as having both warnings is really weird. Furthermore, if you use an IDE, errors are shown together in the code.